test/test_images/grpc-ping/main.go

```diff
@@ -31,12 +32,20 @@ func pong(req *ping.Request) *ping.Response {
 	return &ping.Response{Msg: req.Msg + os.Getenv("SUFFIX")}
 }
 
+func pongHostname(req *ping.Request) *ping.Response {
+	host, err := os.Hostname()
+	if err != nil {
+		log.Println("os.Hostname() returning an error:", err)
+	}
+	return &ping.Response{Msg: req.Msg + host}
+}
+
 type server struct{}
 
 func (s *server) Ping(ctx context.Context, req *ping.Request) (*ping.Response, error) {
 	log.Printf("Received ping: %v", req.Msg)
-
-	resp := pong(req)
+	time.Sleep(1 * time.Second)
```

Comment:
what is this sleep for?

Reply:
Without delay in response GRPC app does not scale.

Reply:
Make this an option :) e.g.extend gRPC API to haven an `optional sleepSecs int = 2;`

Reply:
Or env var for the image.

Reply:
Okay I can update the app to make it optional